client-go/env/local.env

```diff
@@ -0,0 +1,3 @@
+export JWT_SECRET=secret
+export PUBSUB_EMULATOR_HOST=localhost:8681
+export PUBSUB_PROJECT_ID=my-project-id
```

Comment:
Duplicated this file from the `service-go` so breaking DRY. Better approach would be to consider sharing the config struct and test configs between `client-go` and `service-go` but that has a different set of concerns.